File: Ruby/TestP3.rb
#encoding utf-8

require_relative 'Game'
require_relative 'controller'
require_relative 'textUI'


class TestP3
    def main
        n_players = 3
        game = Irrgarten::Game.new(n_players)
        view = UI::TextUI.new
        controller = Control::Controller.new(game,view)
        controller.play
    end
end

test = TestP3.new
test.main

File: Java/Irrgarten/src/tests/TestP3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package tests;

import irrgarten.Game;
import irrgarten.UI.UI;
import irrgarten.UI.TextUI;
import irrgarten.controller.Controller;



/**
 *
 * @author mariorgzlpz
 */
public class TestP3 {
    public static void main(String[] args) {
        int nplayers = 3;
        Game game = new Game(nplayers);
        UI view = new TextUI();
        Controller controller = new Controller(game,view);
        controller.play();
    }
}
